src/components/shared/LibraryTypeSelect.vue

```diff
@@ -20,8 +20,8 @@ import { filterByAttribute, mapAttribute } from '@/api/JsonApi'
 // select elements can't handle the former. So we encode it
 const UNDEFINED = '_undefined'
 
-const encode = (value) => (value === undefined ? UNDEFINED : value)
-const decode = (value) => (value === UNDEFINED ? undefined : value)
+const encode = (value) => (value === undefined ? 'UNDEFINED' : value)
```

Comment:
Why is undefined in quotes here?

Reply:
Good finding. It's a typo, now changed 👍